app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/addedit/VaultAddEditViewModel.kt

```diff
@@ -1567,25 +1570,34 @@ class VaultAddEditViewModel @Inject constructor(
     }
 
     private fun VaultAddEditState.determineContentState(
-        vaultData: VaultData,
+        vaultData: VaultData?,
         userData: UserState?,
     ): VaultAddEditState {
+        val internalVaultData = vaultData
+            .takeUnless { it == null }
```

Comment:
`takeUnless { it == null }` is not necessary. Could just be `vaultData ?: VaultData(...)`